### DISCUSSION TITLE
Add a "copy" option to the macOS share menu

### DISCUSSION THREAD

**Discussion by jacksonwelsh** [ciderapp/Cider#916] [Added Features]
Cider doesn't have an option to copy a share link in the context menu for songs and albums.

![jacksonwelsh - Cider 2022-04-22 at 13 04 14](https://user-images.githubusercontent.com/20214768/164770071-a8106796-28ec-4d61-878a-c3fb94f82d7b.png)

**Comment by booploops** [comment]
Resolved in newest develop

**Comment by Amaru8** [comment]
Locking discussion.
